skimage/_build.py

```diff
@@ -35,8 +38,7 @@ def cython(pyx_files, working_path=''):
     except ImportError:
         # If cython is not found, the build will make use of
         # the distributed .c files if present
-        c_files = [f.replace('.pyx.in', '.c').replace('.pyx', '.c') for f in pyx_files]
-        for cfile in [os.path.join(working_path, f) for f in c_files]:
+        for cfile in abs_c_files:
```

Comment:
These changes are the subject of this PR #3311 . It has been in the works for a while, but keeps slipping as there are always more pressing issues. Is it necessary for your PR?

Reply:
nevermind, this is a different thing.